src/content/en/2021/javascript.md

```diff
@@ -16,18 +16,19 @@ featured_stat_2: 45.6%
 featured_stat_label_2: Images are the most requested content type on pages loaded by JavaScript.
 featured_stat_3: 2.7%
 featured_stat_label_3: Percent of desktop pages using custom elements.
-unedited: true
 ---
 
 ## Introduction
 
-The speed and consistency that the JavaScript language has evolved with over the past years is tremendous. While in the past it was used primarily for client-side, it has taken a very important and respected place in the world of building services and server-side tools. JavaScript has evolved to a point where it is not only possible to create faster applications but also to <a hreflang="en" href="https://blog.stackblitz.com/posts/introducing-webcontainers/">run servers within browsers</a>.
+The speed and consistency in which the JavaScript language has evolved with over the past years is tremendous. While in the past it was used primarily for client-side, it has taken a very important and respected place in the world of building services and server-side tools. JavaScript has evolved to a point where it is not only possible to create faster applications but also to <a hreflang="en" href="https://blog.stackblitz.com/posts/introducing-webcontainers/">run servers within browsers</a>.
 
 There is a lot that happens in the browser when rendering the application, from downloading JavaScript to parsing, compiling and executing it. Let's start with that first step and try to understand how much JavaScript is actually requested by pages.
 
 ## How much JavaScript do we load?
 
-To measure is the key towards improvement, they say. To improve the usage of JavaScript in our applications, we need to measure how much JavaScript being shipped is actually required. Let's dig in to understand the distribution of JavaScript bytes per page, considering what a major role it plays in the web setup.
+To measure is the key towards improvement, they say. To improve the usage of JavaScript in our applications, we need to measure how much of the JavaScript being shipped is actually required. Let's dig in to understand the distribution of JavaScript bytes per page, considering what a major role it plays in the web setup.
+
+{# TODO - "To measure is the key towards improvement, they say." - is this a quotation or a reference to something? the phrasing implies it #}
```

Comment:
@NishuGoel

Reply:
It's a reference to [this quote](https://www.goodreads.com/quotes/632992-measurement-is-the-first-step-that-leads-to-control-and) and variations of it. Didn't credit the quote author per se as its not really the usage of that quote but understanding in general.

Reply:
@shantsis Hey, instead of 
"they say that, to measure...", could you put it as:

They say, "To measure is the key towards..."

Just sounds a bit better.

Reply:
Done!